homework/02LessonTwo.js

```diff
@@ -117,7 +158,20 @@ function whichSpecies(character) {
 // return the string "divisible by 2" when number % 2 === 0
 // return the string "divisible by 3" when number % 3 === 0
 // return the string "divisible by 5" when number % 5 === 0
-
+function testNumber(number) {
+  if(number % 4 === 0){
+    return "divisible by 4";
+  }
+  else if (number % 2 === 0){
+    return "divisible by 2";
+  }
+  else if (number % 3 === 0){
+    return "divisible by 3";
+  }
+  else if (number % 5 === 0){
+    return "divisible by 5";
+  }
+}
 
```

Comment:
All your mocha tests passed! Bravo.

![bravo](http://www.picgifs.com/reaction-gifs/reaction-gifs/good-job-succes-nailed-it/good-job44.gif)